windows/web/slimevr_web_installer.nsi

```diff
@@ -645,6 +676,7 @@ SectionEnd
 
 LangString DESC_SEC_SERVER ${LANG_ENGLISH} "Installs latest SlimeVR Server."
 LangString DESC_SEC_JRE ${LANG_ENGLISH} "Downloads and copies Java JRE 17 to installation folder. Required for SlimeVR Server."
+LangString DESC_SEC_WEBVIEW ${LANG_ENGLISH} "Downloads and install Webview2 if not already installed. Required for the SlimeVR GUI"
```

Comment:
Shouldn't be disable-able, I feel like since it's **required** anyway.

Reply:
i mean ... you can disable the server ....

Reply:
fair